myapp/spec/system/task_spec.rb

```diff
@@ -0,0 +1,186 @@
+require 'rails_helper'
+
+RSpec.describe Task, type: :system do
+  let(:task) { create(:task) }
+
+  describe 'タスク表示' do
+    context '一覧' do
+      it 'DBに保存されたデータがあれば一覧表示' do
+        task_list = create_list(:task, 3)
+        visit root_path
+        expect(page).to have_content task_list[0].id
```

Comment:
上２行と３行目以降が「テストの準備」「テスト対象の実行」「テスト結果の確認」と、文脈の異なるまとまりになるので、そのような単位で改行を入れて欲しいです。
そうすることで、他の人やｎヶ月後の自分が見たときに処理のまとまりを理解しやすくなります。
```suggestion
        task_list = create_list(:task, 3)

        visit root_path
        
        expect(page).to have_content task_list[0].id
```

Reply:
空行を挿入して処理内容ごとに分割しました